Common/src/main/java/com/tabnineCommon/binary/requests/capabilities/CapabilitiesResponse.kt

```diff
@@ -7,4 +7,22 @@ import com.tabnineCommon.capabilities.Capability
 data class CapabilitiesResponse(
     @SerializedName("enabled_features")
     var enabledFeatures: List<Capability>? = null,
+    @SerializedName("experiment_source")
+    var experimentSource: ExperimentSource? = null
 ) : BinaryResponse
+
+enum class ExperimentSource {
+    @SerializedName("API")
+    API,
+
+    @SerializedName("APIErrorResponse")
+    APIErrorResponse,
+
+    @SerializedName("Hardcoded")
```

Comment:
I think you don't need the attribute if it the same

Reply:
I prefer keeping, just to be explicit, enums are weird :}